src/commands/appSettings/connectionSettings/eventHubs/IEventHubsConnectionWizardContext.ts

```diff
@@ -3,23 +3,17 @@
  *  Licensed under the MIT License. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
-import type { AuthorizationRule, EHNamespace } from "@azure/arm-eventhub";
-import type { ResourceGroup } from "@azure/arm-resources";
+import type { ResourceManagementModels } from "@azure/arm-resources";
 import { ISubscriptionContext } from "@microsoft/vscode-azext-utils";
 import { EventHubsConnectionTypeValues, StorageConnectionTypeValues } from "../../../../constants";
 import { ISetConnectionSettingContext } from "../ISetConnectionSettingContext";
 
 export interface IEventHubsConnectionWizardContext extends ISetConnectionSettingContext, Partial<ISubscriptionContext> {
-    resourceGroup?: ResourceGroup;
+    resourceGroup?: ResourceManagementModels.ResourceGroup;
 
     // Connection Types
     azureWebJobsStorageType?: StorageConnectionTypeValues;
     eventHubsConnectionType?: EventHubsConnectionTypeValues;
 
-    // Netherite
-    newEventHubsNamespaceName?: string;
-    eventHubsNamespace?: EHNamespace;
-    newAuthRuleName?: string;
-    authRule?: AuthorizationRule;
-    newEventHubName?: string;
+    // Netherite uses all of the eventhub namespace settings in IEventHubWizardContext
```

Comment:
Probably okay, but I'll double check some `Netherite` scenarios to make sure this doesn't break.  We'll probably want CTI to pay extra attention to this flow when they do their test pass

More reason for me to get some tests in for Durable as well...